test/ParamTests.cpp

```diff
@@ -0,0 +1,32 @@
+#include <gtest/gtest.h>
+#include <rccl/rccl.h>
+
+#include "param.h"
+
+namespace RcclUnitTesting
+{
+    TEST(ParamTests, initEnv_ParseValidConfFile){
+        // This function call reads and opens the conf file from the path
+        // which is set using env. variable NCCL_CONF_FILE
+        initEnv();
+
+        EXPECT_EQ(getenv("TEST_VAR_WITH_NO_VALUE"), nullptr);
+        EXPECT_STREQ(getenv("TEST_VAR"), "12345"); 
+        
+        // Clean up
+        unsetenv("TEST_VAR_WITH_NO_VALUE");
+        unsetenv("TEST_VAR");
```

Comment:
You can remove these lines too since you are not using `setenv` for setting up these env vars.